### ISSUE TITLE
`onScroll` called when screen is loaded only on Android

### ISSUE THREAD

**Issue by KrisLau** [react-native-webview/react-native-webview#2751]
**Bug description:**
When the screen with a WebView is loaded, `onScroll` called immediately but only on Android. I have a function which checks for the user scroll to know whether or not to collapse the header. I use onScrollEndDrag for other components but webview doesn't have that.

Current workaround:
If the `e.nativeEvent.velocity.x === e.nativeEvent.velocity.y` then it is being triggered by the initial load 

**To Reproduce:**
Create a webview log onScroll events. Notice that it is triggered on load

**Expected behavior:**
Only trigger when user scrolls

**Screenshots/Videos:**
N/A

**Environment:**
 - OS: Pixel 4
 - OS version: Android 11
 - react-native version: 0.70.6
 - react-native-webview version: 11.23.1

**Comment by TheAlmightyBob** [comment]
I just tested this with the `react-native-webview` example app (there's a section dedicated to testing the scroll event) and saw the opposite behavior to what you describe. iOS fired a scroll event on initial load but Android did not.

I did see that there appears to be an [alternate method of listening for scroll events on Android](https://developer.android.com/reference/android/view/View.OnScrollChangeListener), but there's no point trying that if Android already has the desired behavior.
